.genx/templates/web-components/component/component.gridOptions.hbs

```diff
@@ -0,0 +1,5 @@
+import { getDateFormatter, getNumberFormatter{{#if route.FDC3EventHandlersEnabled}}, sendEventOnChannel{{/if}} } from '../../../utils';
+
+{{#if tile.config.gridOptions}}
```

Comment:
We should add types to all these configs (coldefs, form schemas and grid options) i.e.

`export const columnDefs: ColDef[]` etc.

Reply:
[68a6087](https://github.com/genesiscommunitysuccess/blank-app-seed/pull/275/commits/68a6087374efa86b795123814f3f9e282506a562)